src/ray/raylet/scheduling/policy/bundle_scheduling_policy.h

```diff
@@ -0,0 +1,127 @@
+// Copyright 2021 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <vector>
+
+#include "ray/raylet/scheduling/policy/scheduling_context.h"
+#include "ray/raylet/scheduling/policy/scheduling_policy.h"
+#include "ray/raylet/scheduling/policy/scorer.h"
+
+namespace ray {
+namespace raylet_scheduling_policy {
+
+class BundleSchedulingPolicy : public ISchedulingPolicy {
```

Comment:
Do we have sufficient unit test for these policies?

Reply:
> Do we have sufficient unit test for these policies?

These codes comes from gcs_resource_scheduler, this PR aims to remove gcs_resource_scheduler, so I just move the logic inside gcs_resource_scheduler to scorer.h/.cc and bundle_scheduling_policy.h/.cc and did not add any new unit test.

The original `gcs_resource_scheduler_test` (now rename to `cluster_resource_scheduler_2_test`) has some unit test related to these policies.